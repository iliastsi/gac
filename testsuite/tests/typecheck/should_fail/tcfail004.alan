arrayParamErr() : proc
  foo(foo : byte []) : proc
  {
    writeString(foo);
  }
{
  foo("42\n");
}
