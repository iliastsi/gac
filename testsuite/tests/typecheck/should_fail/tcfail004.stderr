tcfail004.alan:2:7-19: Error: Type mismatch at `foo :  byte[]'
	Array parameters have to be passed by reference

